### ISSUE TITLE
Support Kubernetes v1.6.6

### ISSUE THREAD

**Issue by seanknox** [Azure/acs-engine#797]

**Comment by JackQuincy** [comment]
https://github.com/Azure/acs-engine/pull/796

**Comment by dcieslak19973** [comment]
I believe I was able to create a k8s v1.6.6 cluster by setting orchestratorProfile:

`"orchestratorProfile": {
      "orchestratorType": "Kubernetes",
      "orchestratorVersion": "1.6.6"
    },
`
I did a bit more than the basic, trying to put the cluster in a custom VNET, using that template and adding the orchestratorVersion line, essentially, as the difference.

The cluster provisioned (nit - it created a Public IP, which I would prefer it wouldn't do given I specified the custom VNET, but deleting internet-facing things is OK at this point). When I SSH into the k8s master, I can run "kubectl version" and get:

`k8s-master-30822299-0:~$ kubectl version
Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.6", GitCommit:"7fa1c1756d8bc963f1a389f4a6937dc71f08ada2", GitTreeState:"clean", BuildDate:"2017-06-16T18:21:54Z", GoVersion:"go1.7.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.6", GitCommit:"7fa1c1756d8bc963f1a389f4a6937dc71f08ada2", GitTreeState:"clean", BuildDate:"2017-06-16T18:21:54Z", GoVersion:"go1.7.6", Compiler:"gc", Platform:"linux/amd64"}
`

So far, so good.

I run kubectl proxy on my local box and then attempt to connect to the https://localhost:8001/ui only to get:

`{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "no endpoints available for service \"kubernetes-dashboard\"",
  "reason": "ServiceUnavailable",
  "code": 503
}`

When I go back to the master node and run:

`kubectl get pods --namespace=kube-system | grep dashboard`

I get:

`kubernetes-dashboard-3868706175-84mh9           0/1       CrashLoopBackOff   227        20h`

Getting logs for this pod yields"

`Using HTTP port: 9090
Creating API server client for https://10.0.0.1:443
Error while initializing connection to Kubernetes apiserver. This most likely means that the cluster is misconfigured (e.g., it has invalid apiserver certificates or service accounts configuration) or the --apiserver-host param points to a server that does not exist. Reason: Get https://10.0.0.1:443/version: dial tcp 10.0.0.1:443: i/o timeout
Refer to the troubleshooting guide for more information: https://github.com/kubernetes/dashboard/blob/master/docs/user-guide/troubleshooting.md
`
